planner_cspace/src/dummy_robot.cpp

```diff
@@ -55,9 +57,24 @@ class DummyRobot
   }
   void cbInit(const geometry_msgs::PoseWithCovarianceStamped::ConstPtr &msg)
   {
-    x_ = msg->pose.pose.position.x;
-    y_ = msg->pose.pose.position.y;
-    yaw_ = tf::getYaw(msg->pose.pose.orientation);
+    tf::StampedTransform trans_msg_odom;
+    try
+    {
+      tfl_.lookupTransform(msg->header.frame_id, "odom", ros::Time(0), trans_msg_odom);
```

Comment:
`waitForTransform` should be added since `msg->header.frame_id` to `"odom"` transform is not always static.